src/main/resources/application.yml

```diff
@@ -114,5 +114,8 @@ application:
     system-user-capabilities:
       retry-attempts: ${KAFKA_SYS_USER_CAPABILITIES_RETRY_ATTEMPTS:60}
       retry-delay: ${KAFKA_SYS_USER_CAPABILITIES_RETRY_DELAY:5s}
+  federated-auth:
+    enabled: ${SINGLE_TENANT_UX_ENABLED:false}
```

Comment:
In acceptance criterias this flag is specified without prefix ENABLED, also in readme it also without this prefix currently. Similar true|false flags are also without this prefix, so let's keep the same convention and use just SINGLE_TENANT_UX